tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/variables/VariableVisibilityTest.java

```diff
@@ -151,6 +152,18 @@ public void controlFlowVariableVisibilityTest() throws BallerinaTestException {
         localVariables = debugTestRunner.fetchVariables(debugHitInfo.getRight(), DebugTestRunner.VariableScope.LOCAL);
         Assert.assertEquals(localVariables.size(), 47);
 
+        // visibility test of local variable defined inside `if` statement
+        debugTestRunner.resumeProgram(debugHitInfo.getRight(), DebugTestRunner.DebugResumeKind.NEXT_BREAKPOINT);
+        debugHitInfo = debugTestRunner.waitForDebugHit(10000);
+        localVariables = debugTestRunner.fetchVariables(debugHitInfo.getRight(), DebugTestRunner.VariableScope.LOCAL);
+        Assert.assertEquals(localVariables.size(), 47);
```

Comment:
We already have variable visibility test inside `if` statement. Please refer [here](https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/variables/VariableVisibilityTest.java#L121). Is there any specific reason to add again here?

Reply:
Yes, it is for demonstrate the visibility of variable which is defined inside an if statement (or any other blocks or loops).